Formula/llvm.rb

```diff
@@ -21,47 +21,73 @@ class Llvm < Formula
   desc "Next-gen compiler infrastructure"
   homepage "http://llvm.org/"
 
+  llvm_version = "3.8.0"
```

Comment:
I've got to be honest, I _hate_ formulae adding this sort of thing because it makes URLs from inside formulae impossible to just copy/paste.